homeworks/app/src/components/common/Page.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "../../styles/page.css"
+
+class Page extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { 
+      header: props.headertext ? props.headertext : ""
```

Comment:
Please add PropTypes for checking props https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes